### ISSUE TITLE
running modules individually

### ISSUE THREAD

**Issue by yaboy1** [zone117x/node-cryptonote-pool#233]
Curious as to how the modules work when running individually. Couple quick questions please:

1. Is running the 4 modules (pool, api, unlocker, payments) separately in different terminals under the same login the same as launching "node init.js" in one terminal (kicks off all 4) from a functionality perspective?

2. Can I run individual modules and be ok? (pool definitely must run)

Why I'm asking I'd like to feel confident in knowing if I run just pool and api that everything is good to go. I have this setup running beautifully at the moment (thank you for everything). 

I am using it where only my machines are connecting to it in a LAN so I really only need the pool to service my miners and the api to look at things. The miners all use the same wallet address (which is the same as the pool's wallet address). So the functionality of unlocker and payments is not needed I believe.

Is this setup functionally sound?

Please let me know what you think! And again thank you for all you do.

**Comment by p1vynodejs** [comment]
yes, it will be okay. and also dont forget to disable unlocker and payments in your config.json.

**Comment by yaboy1** [comment]
sweet thank you, I have done that. final question: how can i confirm the results of the pool mining node is properly submitting its results and stats to the network? the monero daemon states mining is zero which i assume is because i am not mining using the daemon, but then that makes me wonder how to check that my hash rate is a part of the network's total hash? i hope that makes sense! basically i just dont want to wait for weeks only to find out something wasn't talking right... thank you!!

**Comment by yaboy1** [comment]
things like the "total miners paid" being a -1 on the stats page of the api is what bothers me a little haha. speaking of which, is there a list of all the api calls? again much thanks peeps
